app/src/webkit/java/org/mozilla/focus/webkit/WebkitView.java

```diff
@@ -396,7 +399,28 @@ public void onDownloadStart(String url, String userAgent, String contentDisposit
                 if (downloadCallback != null) {
                     final String name = URLUtil.guessFileName(url, contentDisposition, mimetype);
                     final Download download = new Download(url, name, userAgent, contentDisposition, mimetype, contentLength, false);
-                    downloadCallback.onDownloadStart(download);
+                    AlertDialog.Builder localBuilder = new AlertDialog.Builder(getContext());
+                    localBuilder.setTitle(R.string.download_notification_title );
+                    localBuilder.setMessage(R.string.download_notification_message );
+                    localBuilder.setPositiveButton(R.string.action_ok , new DialogInterface.OnClickListener()
+                    {
```

Comment:
please run checkstyle for the format.
Another reason this should be in the UI layer: We might need permission before showing this prompt.